Updating text input, textarea, and select docs (#1136)
hashicorp/design-system: website/docs/components/form/textarea/partials/code/how-to-use.md

</Hds::Form::Textarea::Field>
```

#### Validation

> To indicate a field is invalid, declare that it’s invalid by using the `@isInvalid` argument and provide an error message using the `Error` contextual component.

Reviewer: ```suggestion
To indicate a field is invalid, use `@isInvalid` argument and provide an error message using the `Error` contextual component.
```

#### HTML native attributes

> This component supports use of `...attributes`. This means you can use all the standard HTML attributes of the `<textarea>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (e.g., providing a `name` for the control, or adding `min` `max` `minlength` `maxlength` `pattern` attributes to it).

Reviewer: ```suggestion
This component supports use of `...attributes`. This means you can use all the standard HTML attributes of the `<textarea>` element. This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (e.g., providing a `name` for the control, or adding `min`, `max`, `minlength`, `maxlength`, or `pattern` attributes to it).
```
